collections/sentinel-3-l1b-olci.yaml

```diff
@@ -154,222 +154,251 @@ Summaries:
         value:
-          - 300
-          - 300
-        unit: m
-    - name: B11
+          - 0.002976190476190
+          - 0.002976190476190
+        unit: degree
+    - description: Chlorophyll fluorescence baseline, red edge transition
+      name: B11
       center_wavelength: 0.70875
       full_width_half_max: 0.01
       openeo:gsd:
         value:
-          - 300
-          - 300
-        unit: m
-    - name: B12
+          - 0.002976190476190
+          - 0.002976190476190
+        unit: degree
+    - description: O2 absorption / clouds, vegetation
+      name: B12
       center_wavelength: 0.75375
       full_width_half_max: 0.0075
       openeo:gsd:
         value:
-          - 300
-          - 300
-        unit: m
-    - name: B13
+          - 0.002976190476190
+          - 0.002976190476190
+        unit: degree
+    - description: O2 absorption / aerosol correction
+      name: B13
       center_wavelength: 0.76125
       full_width_half_max: 0.0025
       openeo:gsd:
         value:
-          - 300
-          - 300
-        unit: m
-    - name: B14
+          - 0.002976190476190
+          - 0.002976190476190
+        unit: degree
+    - description: Atmospheric correction
+      name: B14
       center_wavelength: 0.764375
       full_width_half_max: 0.00375
       openeo:gsd:
         value:
-          - 300
-          - 300
-        unit: m
-    - name: B15
+          - 0.002976190476190
+          - 0.002976190476190
+        unit: degree
+    - description: O2 absorption used for cloud top pressure, fluorescence over land
+      name: B15
       center_wavelength: 0.7675
       full_width_half_max: 0.0025
       openeo:gsd:
         value:
-          - 300
-          - 300
-        unit: m
-    - name: B16
+          - 0.002976190476190
+          - 0.002976190476190
+        unit: degree
+    - description: Atmospheric / aerosol correction
+      name: B16
       center_wavelength: 0.77875
       full_width_half_max: 0.015
       openeo:gsd:
         value:
-          - 300
-          - 300
-        unit: m
-    - name: B17
+          - 0.002976190476190
+          - 0.002976190476190
+        unit: degree
+    - description: Atmospheric / aerosol correction, clouds, pixel co-registration
+      name: B17
       center_wavelength: 0.865
       full_width_half_max: 0.02
       openeo:gsd:
         value:
-          - 300
-          - 300
-        unit: m
-    - name: B18
+          - 0.002976190476190
+          - 0.002976190476190
+        unit: degree
+    - description: Water vapour absorption reference. Common reference band with SLSTR. Vegetation monitoring
+      name: B18
       center_wavelength: 0.885
       full_width_half_max: 0.01
       openeo:gsd:
         value:
-          - 300
-          - 300
-        unit: m
-    - name: B19
+          - 0.002976190476190
+          - 0.002976190476190
+        unit: degree
+    - description: Water vapour absorption, vegetation monitoring (maximum REFLECTANCE)
```

Comment:
I know in our SH docs "REFLECTANCE" is all caps but I would use "reflectance" here. I don't see the need to have this in all caps.

Reply:
Thanks, I agree. Changed.